index.tsx

```
@@ -1,6 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+//dhsaklnfa
+
+//hdsafnas
+
+
+// hfhaskfdhnas1234
```

ESLint

[error] More than 1 blank line not allowed.

(no-multiple-empty-lines)